app/Jobs/SendPostToCustomerIo.php

```diff
@@ -2,60 +2,67 @@
 
 namespace Rogue\Jobs;
 
-use DoSomething\Gateway\Blink;
 use Illuminate\Bus\Queueable;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Foundation\Bus\Dispatchable;
 use Illuminate\Queue\InteractsWithQueue;
 use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Redis;
 use Rogue\Models\Post;
+use Rogue\Services\CustomerIo;
 
 class SendPostToCustomerIo implements ShouldQueue
 {
     use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
 
     /**
-     * The post to send to Customer.io.
+     * Delete the job if its models no longer exist.
      *
-     * @var Post
+     * @var bool
      */
-    protected $post;
+    public $deleteWhenMissingModels = true;
 
     /**
-     * Whether or not to log that this Post was sent to Customer.io, via Blink.
+     * The post to send to Customer.io.
      *
-     * @var bool
+     * @var Post
      */
-    protected $log;
+    protected $post;
 
     /**
      * Create a new job instance.
      *
      * @return void
      */
-    public function __construct(Post $post, $log = true)
+    public function __construct(Post $post)
```

Comment:
We'd previously added this `$log` parameter to reduce log volume when running scripts. Now that we're on a shared Papertrail account (rather than per-app add-ons), this is not a significant usage concern.